docs/intro.md

```diff
@@ -227,6 +231,303 @@ Please refer to the [project
+Some data sources also expect environment variables that provide data on network
+parameters used by the Cardano main chain:
+- `CARDANO_SECURITY_PARAMETER`: the *k* parameter that determines how many
+  confirmations are required for a block to be considered stable. Needs to be set
+  to the value of `securityParam` as configured in the shelley-genesis config file
+  passed to `cardano-node` via the `--shelley-genesis` parameter,
+- `CARDANO_ACTIVE_SLOTS_COEFF`: The fraction of slots in which a block is expected
+  to be produced on Cardano. Needs to be set to the value of `activeSlotsCoeff` as
+  configured in the shelley-genesis config file passed to `cardano-node` via the
+  `--shelley-genesis` parameter,
+
+as well as information on Cardano slot and epoch configuration:
+
+- `MC__SLOT_DURATION_MILLIS`: duration of a Cardano slot in millis
+- `MC__EPOCH_DURATION_MILLIS`: duration of a Cardano epoch in millis
+- `MC__FIRST_EPOCH_TIMESTAMP_MILLIS`, `MC__FIRST_EPOCH_NUMBER` and
+  `MC__FIRST_SLOT_NUMBER`: these parameters must indicate the point in time after
+  the start of the Shelley Cardano era, after which the two other parameters above
+  have not changed anymore. Typically, these values should point to the beginning
+  of the Shelley era, but for chains that modified their slot or epoch duration
+  after that, the start of any Cardano epoch after the last change could be used,
+  as long as it is before any Partner Chains smart contracts have been deployed
+
+Lastly, every Partner Chain node needs the `BLOCK_STABILITY_MARGIN` variable to
+be set. This variable determines an additional margin added to
+`CARDANO_SECURITY_PARAMETER` when selecting the latest stable Cardano block to be
+included in the block header. This value can be set to `0` by default, but higher
+values if the network experiences a high number of blocks rejected because of
+Db-Sync lag.
+
+##### Genesis configuration environment variables
+
+Some Partner Chains may choose to read genesis configuration for some pallets from
+the environment when running a node without a chain spec, or generating a new spec.
+The list of these pallets and their environment variables is presented below. The
+names of the enviroment variables correspond directly to the names of genesis
+config fields.
+
+###### Sidechain
+- `GENESIS_UTXO`
+
+###### Committee selection
+- `COMMITTEE_CANDIDATE_ADDRESS`: 
+- `D_PARAMETER_POLICY_ID`
+- `PERMISSIONED_CANDIDATES_POLICY_ID`
+
+###### Native Token Management
+- `NATIVE_TOKEN_POLICY_ID`
+- `NATIVE_TOKEN_ASSET_NAME`
+- `ILLIQUID_SUPPLY_VALIDATOR_ADDRESS`
+
+#### Keys
+The partner chain node needs three different keys to be present in a Substrate key store accessible
+by the partner chain node.
+1. `sidechain_pub_key`: The sidechain public key (ECDSA)
+1. `aura_pub_key`: The aura public key (sr25519)
+1. `grandpa_pub_key`: The grandpa public key (Ed25519)
+
+By default the partner chain node process will look for key stores in the base path directory.
```

Comment:
Not really. This is not our toolkit business. As far as I'm concerned logic of resolving keystore location is hardcoded in substrate and it looks in `<base_dir>/chains/<chain_id>`
where `chain_id` is taken from the used chain spec.